src/api.ts

```diff
@@ -355,6 +355,12 @@ export interface ResultSet {
      * table.
      */
     lastInsertRowid: bigint | undefined;
+
+    /** Converts the result set to JSON.
+     *
+     * This is used automatically by `JSON.stringify()`, but you can also call it explicitly.
+     */
+    toJSON(): any;
```

Comment:
Does Row need one as well, if someone were to just try to serialize a row and not the entire ResultSet?

Reply:
Right, I understand that serializing ResultSet works fine. The case in question is a single independent row that could have a bigint. So, if I write some code to enable bigints in results, then makes a query iterates rows and attempts to serialize each one of them independently, it seems like we could run into this again.

Reply:
Yes, that's true. But implementing the `toJSON()` property for `Row` is very awkward, for the reasons explained above.

Reply:
I don't think it necessarily needs a by-name conversion of the values.  An array is just fine.

Reply:
So you think that there should be no way to access columns by name, or that the user should use a method (`row.get("title")` instead of `row.title`)?

Reply:
For a JSON serialization, I don't think it matters. The data should just all be there for someone to decide how they want to reconstitute.

Reply:
It matters because TypeScript won't allow us to add a `toJSON()` method, because of the `[name: string]: Value;` declaration, which is needed to support access to columns by name (like `row.title`).

Reply:
OK, if you're saying it just can't be done with the current definition, then that's that.